### ISSUE TITLE
[Alpha diversity] page specific updates

### ISSUE THREAD

**Issue by wasade** [biocore/microsetta-interface#94]
The verbiage like “compute” and “Evolutionary breadth” is very technical sounding and can be simplified. Also reduce the number of times that “compute” is used. 

No one understands the chart. Can this be switched to showing histograms? Can the tools on the image be removed? Specific comments:

* “I think it is a look at US/UK over time with a possible snapshot of my data, maybe?”
* Add a title to the chart
* Both Axis need labels
* More of the paragraph needs to be spent on how to interpret the data and what it means and less on how it was derived. That can be put in the learn more section
* Not sure about the tools that pop up out of nowhere when you hover and their purpose. Make them stick and explain purpose or remove.

**Comment by wasade** [comment]
@gwarmstrong, can you take this?

**Comment by gwarmstrong** [comment]
What is the priority of this vs. sample highlighting on empress?

Also, it might make more sense for @dhakim87 to do the plotting work since it is all in plotly and I think he is more familiar. I have not actually used it before.

This could probably be split into two issues, one for language fixes and the other for the plotting.

**Comment by wasade** [comment]
Higher priority. Can the empress stuff be handed to Kalen or Yoshiki?

**Comment by wasade** [comment]
Additional comments from Justin

	- i suggest to hyphenate all instances of alpha-diversity and beta-diversity
	- i suggest to replace instances of 'compute' and 'computing' with 'calculate' and 'calculating', for consistency
	- i suggest to add axes labels to the figure
	- in the plots, i suggest to change 'me' to 'you'

**Comment by gwarmstrong** [comment]
> Can the empress stuff be handed to Kalen or Yoshiki?

The primary blocker is a new endpoint on the public API that returns a sample's features in a format that matches the the collapsed taxonomy for the empress endpoint.

**Comment by wasade** [comment]
Okay thanks. It might be nice to get some other developers familiar with modifying that codebase?

**Comment by gwarmstrong** [comment]
> It might be nice to get some other developers familiar with modifying that codebase?

Agreed. Though I am not sure https://github.com/biocore/microsetta-public-api/issues/100 would be the best starting point for a developer new to the project, since it requires a decent understanding of the code base and the task is a little open ended.

**Comment by wasade** [comment]
Fair, perhaps a good for pair programming target?
